### PR TITLE
Consent json

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Implemented mPower's consent flow more accurately.  
Most of the work was understanding the differences between ResearchStack's consent_section.json parsing vs iOS so I could retrofit iOS' format to work on Android.

I made several stories based on this work, that will need architecture changes, but this will get the consent flow working:
1) https://sagebionetworks.jira.com/browse/MA-12
2) https://sagebionetworks.jira.com/browse/MA-17
3) https://sagebionetworks.jira.com/browse/MA-18
4) https://sagebionetworks.jira.com/browse/MA-19

### CODE REVIEW

**Comment by rianhouston** [comment]
:shipit:

File: MPower/app/src/main/assets/json/consent_section.json [hunk 1]
```diff
@@ -22,117 +22,173 @@
...
       {
-        "identifier": "question3",
-        "prompt": "This app is a research study and not a commercial application.",
-        "positiveFeedback": "This is a research study and not a commercial application.",
-        "negativeFeedback": "This is a research study and not a commercial application.",
+        "identifier": "stressLevel",
+        "prompt": "The survey questions may be stressful for some people.",
         "type": "boolean",
-        "expectedAnswer": "true"
+        "expectedAnswer": true
```
Comment (rianhouston) [inline MPower/app/src/main/assets/json/consent_section.json]: Will this cause issues changing the data type from a string to a boolean?

Comment (dephillipsmichael) [inline MPower/app/src/main/assets/json/consent_section.json]: No, it still works the same, its using Gson parsing to make it a String either way in the class QuizQuestion.

Comment (dephillipsmichael) [inline MPower/app/src/main/assets/json/consent_section.json]: But, iOS version uses boolean, so I wanted to keep them consistent

**Pull request merged**